### DISCUSSION TITLE
Can we edit data in SuperSet?

### DISCUSSION THREAD

**Discussion by premsaggar** [apache/superset#15745] [General]
Hi everyone,

Can we edit a particular cell of data in Apache SuperSet? We'd like our users to be able to update some fields easily from the SupserSet UI. 

Thanks so much :)

**Comment by betodealmeida** [comment]
@premsaggar can you give more details about where your data lives? What database are you using?

**Reply by premsaggar** [reply]
Hi @betodealmeida  :),

Our data is currently in a SQL Server database. We can also keep it in a csv file if need be. Our goal is to present a table to the user and have them directly edit fields in the table and save it back to the file or database. Thanks.

**Reply by nytai** [reply]
you can always use the SQL `UPDATE ...` command

**Reply by premsaggar** [reply]
Thanks, that may be too much for our users. Can it be done by editing a cell directly in a table view? Thanks.

**Reply by betodealmeida** [reply]
That's an interesting idea, but unfortunately not currently supported.

If your data is not too big you could store it in a Google Spreadsheet, and then have your users edit by in the spreadsheet UI. You would then be able to query the sheet as a table in Superset using the Google Sheets connector.
